docs/docs/guides/build/assets/asset-selection-syntax.md

```diff
@@ -1,351 +1,112 @@
 ---
-title: 'Asset selection syntax'
+title: "Asset selection syntax"
```

Comment:
@salazarm sorry, the previous iteration of this PR was a bit confusing -- this is the new asset selection syntax doc.